pkgs/development/libraries/glfw/3.x.nix

```diff
@@ -22,11 +22,11 @@ stdenv.mkDerivation rec {
   buildInputs = [ libX11 libXrandr libXinerama libXcursor libXi libXext ]
     ++ lib.optionals stdenv.isDarwin [ Cocoa Kernel ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=ON" ];
-
-  preConfigure  = lib.optional (!stdenv.isDarwin) ''
-    substituteInPlace src/glx_context.c --replace "libGL.so.1" "${lib.getLib libGL}/lib/libGL.so.1"
-  '';
+  cmakeFlags = [
+    "-DBUILD_SHARED_LIBS=ON"
+  ] ++ lib.optional (!stdenv.isDarwin) [
```

Comment:
This should be `lib.optionals` since it's a list.

Reply:
Good find. Should I make a PR to fix this or would you like to push a fix?
What does this mean in practice? What's broken if it's like this?